src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -49,9 +53,27 @@ function sanitizeHtml(htmlText) {
         <div class="htmlpopup-header">{{ feature.title }}</div>
         <div class="htmlpopup-content">
             <div v-for="[key, value] in sanitizedFeatureDataEntries" :key="key" class="mb-1">
+                <div
+                    v-if="disclaimerAgree && value.includes('iframe')"
+                    data-cy="feature-detail-media-disclaimer"
+                    class="py-3 header-warning-dev bg-danger text-white text-center text-wrap text-truncate overflow-hidden fw-bold p-1"
+                >
+                    <div class="pb-2">{{ i18n.t('media_disclaimer') }}</div>
+                    <button
+                        data-cy="feature-detail-media-disclaimer-button"
+                        class="px-3 rounded-2 btn btn-sm btn-light align-items-center"
+                        @click="setDisclaimerAgree"
+                    >
+                        {{ i18n.t('disclaimer_agree') }}
+                    </button>
+                </div>
```

Comment:
Maybe we should have a little discussion among the team about this, and see if that is a "good enough" for everyone or if there would be a cleverer solution that raises from the discussion